src/Calculator.jsx

```diff
@@ -81,10 +81,19 @@ export default function Calculator (props) {
         setErrors({});
     }
 
+    function useSampleData() {
+        if (model.sample_values) {
+          setValues(model.sample_values);
+          setTimeout(function() {
+            compute(model.sample_values);
```

Comment:
For reasons beyond my current understanding, a simple `compute` without forcing the model values would produce the estimation with the _previous_ form values (not the ones updated with `setValues` on line 86), even though visually the form appears to change and displays the new values. If someone has an explanation for this behavior I'm happy to learn more about the reasons and come up with a more elegant solution.